trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import warnings
+from SUAVE.Core import Units
+from matplotlib.pyplot import cm
+
+
+def plot_weight_comparison(lst_files, caseNames=[], units='kg'):
+    if len(caseNames) < len(lst_files):
+        warnings.warn('caseNames and the number of files do not match, adding generic case names')
+        for i in range(len(caseNames), len(lst_files)):
+            caseNames.append("Case " + str(i))
+    if not isinstance(lst_files, list):
+        lst_files = [lst_files]
+    main_weight = {}
```

Comment:
There are many instances of this but I'll just leave one comment here. In general we try to keep variable assignment in SUAVE in table format, so this should be modified so that the equal signs line up.